### PR TITLE
Fix typo in Rust tree traversal

### PR DESCRIPTION
**Pull request by berquist**
Creating trees is also more consistent with the Julia implementation.

### CODE REVIEW

File: contents/tree_traversal/code/rust/tree.rs [hunk 1]
```diff
@@ -69,15 +69,16 @@ fn create_tree(num_row: u64, num_child: u64) -> Node {
...
     dfs_stack(&root);
     println!("Queue BFS:");
     bfs_queue(&root);
-    println!("Recursive PostOrder DFS: ");
+    println!("Recursive post-order DFS:");
     dfs_recursive_postorder(&root);
-    println!("Recursive DFS BTree:");
-    dfs_recursive_inorder_btree(&root);
+    println!("Recursive in-order DFS BTree:");
+    let root_binary = create_tree(3, 2);
```
Comment (Gathros) [inline contents/tree_traversal/code/rust/tree.rs]: Why not just have one tree.

Comment (jiegillet) [reply contents/tree_traversal/code/rust/tree.rs]: If you want to test the in-order, you need a binary tree. The other tree is ternary.

Comment (Gathros) [reply contents/tree_traversal/code/rust/tree.rs]: Just keep the other tree binary too.

Comment (jiegillet) [reply contents/tree_traversal/code/rust/tree.rs]: The Julia code and chapter show examples with ternary, though. Not a big deal to me, but I'm not actually reviewing the code, just answering one question :)

Comment (Gathros) [reply contents/tree_traversal/code/rust/tree.rs]: You're right it, should match the julia code.

**Pull request merged**